src/aitsi/m3spin/commons/interfaces/Variable.java

```diff
@@ -1,4 +1,5 @@
 package aitsi.m3spin.commons.interfaces;
 
 public interface Variable extends Factor {
+    int getId();
```

Comment:
Zastanawiam się, czy takie gettery w ogóle powinny trafiać do interfejsów. Jeśli tak, to warto zrobić to również w pozostałych interfejsach.

Reply:
Jeśli tak, to powinno tutaj trafić `getName()`, a następnie należałoby dodać tego typu gettery do pozostałych interfejsów, bo wydaje mi się, że nie wszędzie są. Co o tym myślicie?

Reply:
W sumie to czemu nie, chyba bardzo tego nie potrzebujemy na tym etapie, ale nie jest to dużo pracy zawsze można dodać